sway-lib-std/src/tx.sw

```diff
@@ -195,6 +196,32 @@ pub fn tx_input_type_from_pointer(ptr: u64) -> u8 {
     }
 }
 
+/// If the input's type is `InputCoin` or `InputMessage`,
+/// return the owner as an Option::Some(owner).
+/// Otherwise, returns Option::None.
+pub fn tx_input_owner(index: u64) -> Option<Address> {
+    let type = tx_input_type(index);
+    let owner_offset = match type {
+        0u8 => {
```

Comment:
Are there constants representing these types that we could use here? Hardcoded integers are a bit harder to to follow.

Reply:
Yeah you're right... Looks like https://github.com/FuelLabs/sway/issues/1816 is still open.

Reply:
Can we match on `const` variables though?  https://github.com/FuelLabs/sway/blob/master/sway-lib-std/src/tx.sw#L49